Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -335,6 +335,20 @@ type AndroidProps = $ReadOnly<{|
    * @platform android
    */
   persistentScrollbar?: ?boolean,
+  /**
+   * Fades out the edges.
+   * The default value is false.
+   *
+   * @platform android
+   */
+  fadingEdgesEnabled?: ?boolean,
```

Comment:
Do we need this Boolean? Or can we just check whether the length is 0 for disabled, non zero for enabled?

Reply:
Yeah, you're right. Fixed.